src/dash-table/components/CellLabel/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {
+    MouseEvent,
+    PureComponent
+} from 'react';
+
+interface IProps {
+    className: string;
+    onClick?: (e: MouseEvent) => void;
+    onDoubleClick?: (e: MouseEvent) => void;
+    value: any;
+}
+
+export default class CellLabel extends PureComponent<IProps> {
```

Comment:
Standalone Label component pulled out of pre-existing CellInput comp -- as before we use this to display non-editable fields, fallback dropdown fields and to optimize inputs while cell is not active.